features/413-display-triggers-after-resubmission/test.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Display correct number of triggers after case is resubmitted
+
+      """
+      When an exception is raised for a case, then resolved, and the case is resubmitted. Then, if the case has triggers, it should display the correct number of triggers on the trigger tab.
+      """
+
+  Background:
+    Given the data for this test is in the PNC
+      And "input-message" is received
+
+  @Must
+  @Parallel
+  Scenario: PNC is updated when there are multiple identical results
+    Given I am logged in as "supervisor"
+			And I view the list of exceptions
```

Comment:
Fomatting doesn't look right here. I think tabs should be replaced with white spaces.

Reply:
Or maybe the other way around. White-spaces should be replaced with tabs. It's for consistency so it might be worth checking other `.feature` files.